sdk/identity/azure-identity/TOKEN_CACHING.md

```diff
@@ -68,7 +68,7 @@ record_json = record.serialize()
 
 #### Silently authenticating a user with AuthenticationRecord and TokenCachePersistenceOptions
 
-Once an app has configured an `InteractiveBrowserCredential`, `DeviceCodeCredential`, or `UsernamePasswordCredential` to persist token data and an `AuthenticationRecord`, it's possible to silently authenticate. This example demonstrates an app setting the `TokenCachePersistenceOptions` and retrieving an `AuthenticationRecord` from the local file system to create an `InteractiveBrowserCredential` capable of silent authentication:
+Once an app has configured an `InteractiveBrowserCredential` or `DeviceCodeCredential`, to persist token data and an `AuthenticationRecord`, it's possible to silently authenticate. This example demonstrates an app setting the `TokenCachePersistenceOptions` and retrieving an `AuthenticationRecord` from the local file system to create an `InteractiveBrowserCredential` capable of silent authentication:
```

Comment:
Please remove the `UsernamePasswordCredential` row from the table at the bottom of this doc too.